ongaku/client.py

```diff
@@ -181,6 +181,48 @@ def from_tanjun(
 
         return cls
 
+    @classmethod
+    def from_lightbulb(
+        cls,
+        client: lightbulb.GatewayEnabledClient,
+        *,
+        session_handler: typing.Type[SessionHandler] = BasicSessionHandler,
+        logs: str | int = "INFO",
+        attempts: int = 3,
+    ) -> Client:
+        """From Lightbulb.
+
+        This supports `client` and `player` [injection](../gs/injection.md) for [Lightbulb](https://github.com/tandemdude/hikari-lightbulb)
+
+        Example
+        -------
+        ```py
+        bot = hikari.GatewayBot(...)
+        client = lightbulb.Client(bot)
+        ongaku_client = ongaku.Client.from_arc(client)
+        ```
+
+        Parameters
+        ----------
+        client
+            Your Gateway client for lightbulb.
+        session_handler
+            The session handler to use for the current client.
+        logs
+            The log level for ongaku.
+        attempts
+            The amount of attempts a session will try to connect to the server.
+        """
+        cls = cls(
+            client._app, session_handler=session_handler, logs=logs, attempts=attempts

```

Comment:
I just added `GatewayEnabledClient.app` as a property so you don't need to go through the private attribute anymore

Reply:
```suggestion
            client.app, session_handler=session_handler, logs=logs, attempts=attempts
```